packages/client/README.md

```diff
@@ -694,16 +694,18 @@ Proxy publishing and subscribing are handled on the network overlay level. This
 ```js
 
 // Open publish proxy to multiple nodes on stream
-await publishingClient.openProxyConnections(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
+await publishingClient.setProxies(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
 
 // Remove publish proxy to multiple nodes on stream
-await publishingClient.closeProxyConnections(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
+await publishingClient.setProxies(stream, [], ProxyDirection.PUBLISH)
```

Comment:
Could the comment or description be something like: "To use nodes "0x11111 and 0x22222 as proxies when subscribing to `stream`"?

And the comment or description about remove something like "To remove some/all proxies, call the same method with a different set of nodes. If the node list is empty, proxies are no longer used for the given stream."